app/views/users/settings.html.erb

```diff
@@ -90,6 +90,35 @@
   <br />
   <br />
 
+  <% if current_user.can_moderate? %>
```

Comment:
Here, could we create a (maybe `<h4>` bounded?) subsection for Moderation settings? That way it's a little easier to skim to the settings you want. Also, could we add `a` in front of each instance of this phrase, for both existing and new settings? It's a little better grammatically:

`weekly digest` => `a weekly digest` (and same for all others on this page?

Thank you @keshavsethi this is looking AMAZING!!! 🎉 🙌